src/controllers/expensesController.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const { StatusCodes, ReasonPhrases } = require('http-status-codes');
+const expensesService = require('../services/expensesService');
+const usersService = require('../services/usersService');
+
+function get(req, res) {
+  const { userId, categories, from, to } = req.query;
+
+  res.status(StatusCodes.OK).send(
+    expensesService.get({
+      userId,
+      categories,
+      from,
+      to,
+    }),
+  );
```

Comment:
The `get` function directly sends the result of `expensesService.get()` without checking if the result is valid or if an error occurred. Consider adding error handling to manage cases where the service might return an error or an empty result.